docs/reference/sql/endpoints/odbc/configuration.asciidoc

```diff
@@ -172,7 +181,71 @@ will be considered by default. Choose _All Files (\*.*)_ from the drop down, if
 image:images/sql/odbc/dsn_editor_security_cert.png[]
 
 [float]
-===== 2.4 Logging parameters
+===== 2.4 Connection parameters
+The connection configuration can further be tweaked by the following parameters, but these should only need to be changed in rare cases.
```

Comment:
Sounds a bit weird "these should only need to be changed", but I have no better alternative.

Reply:
maybe just: `.. be tweaked by the following parameters, which is not usually required except from some rare cases.`, or even without the `except from some rare cases`.

Reply:
Thanks. I've removed everything past the comma, since I doubt that these type of vague formulations make a significant difference in users' behaviour.